modules/ROOT/pages/import-api-specification.adoc

```diff
@@ -2,7 +2,7 @@
 ifndef::env-site,env-github[]
 include::_attributes.adoc[]
 endif::[]
-:page-aliases: 7.5@studio::import-api-secification.adoc
+:page-aliases: 7.5@studio::import-api-specifications.adoc
 
```

Comment:
The page name takes no "s" at the end. ":page-aliases: 7.4@studio::import-api-secification.adoc" is OK, but I don't really understand the function of this "page-aliases". The other changes are LGTM! Thanks!

Reply:
please see https://github.com/mulesoft/docs-studio/pull/376/files#r964042979